docs/crictl.md

```diff
@@ -0,0 +1,160 @@
+CRICTL Client User Guide
+========================
+This document presumes you already have `containerd` and `cri-containerd`
+installed and running.
+
+This document is for developers who wish to debug/inspect and manage their pods,
+containers, and container images.
+
+Before generating issues against this document, `containerd`, `cri-containerd`,
+or `crictl` please make sure the issue has not already been submitted.
+
+## Install crictl
+If you have not already installed crictl please install the version compatible
+with the `cri-containerd` you are using. The current version is specified
+[here](./hack/version). A helper command has been included to install the
+dependencies at the right version for you:
+```shell
+# make install.deps
+```
+## Download and Inspect a Container Image
+```shell
+# crictl pull busybox
+  ...
+# crictl inspecti busybox
+```
+## Run a pod sandbox (using a config file)
+```shell
+# cat sandbox-config.json
+{
+    "metadata": {
+        "name": "nginx-sandbox",
+        "namespace": "default",
+        "attempt": 1,
+        "uid": "hdishd83djaidwnduwk28bcsb"
+    },
+    "linux": {
+    }
+}
+
+# crictl runs sandbox-config.json
+e1c83b0b8d481d4af8ba98d5f7812577fc175a37b10dc824335951f52addbb4e
+# crictl sandboxes
```

Comment:
Can you generate the result with newest `crictl`, this is old.

Reply:
done